templates/telemetry/sync/sync.4.schema.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "mozPipelineMetadata": {
     "expiration_policy": {
-      "delete_after_days": 10000
+      "delete_after_days": 400
```

Comment:
Should sync_v5 also change it's retention policy? https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/493789fed14aa9a063607c5a5ab130a117591f84/schemas/telemetry/sync/sync.5.schema.json#L10